pkgs/applications/science/biology/bftools/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, lib, makeWrapper, fetchurl, unzip, openjdk }:
```

Comment:
You should use `openjre`, not `openjdk` unless bftools need to use java compiler.